client/src/components/UI/card/JobCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { Card, Avatar, Button } from "antd";
+import { IconExternalLink } from "@tabler/icons-react";
+import styles from "./JobCard.module.css";
+
+interface JobCardProps {
+  name?: string;
+  company_name?: string;
+  company_location?: string;
+  description?: string;
+  avatarUrl?: string;
+  employees?: string;
+  level?: string;
+  match?: string;
+  location?: string;
+  department?: string;
+}
+
+const JobCard: React.FC<JobCardProps> = ({
+  name,
+  company_name,
+  company_location,
+  description,
+  avatarUrl,
+  employees,
+  level,
+  match,
+  location,
+  department,
+}) => {
+  const truncatedDescription: string | undefined = description
+    ? typeof description === "string" && description.length > 370
+      ? `${description.slice(0, 370)}...`
+      : description
+    : undefined;
+
+  return (
+    <Card className={styles.card}>
+      <div className={styles["card-content"]}>
+        <div className={styles["job-header"]}>
+          <Avatar className={styles.avatar} src={avatarUrl} />
+          <div className={styles["company-info"]}>
+            <h4 className={styles["company-name"]}>{company_name}</h4>
+            <h5 className={styles["company-location"]}>{company_location}</h5>
```

Comment:
why do you call the classes like this and not `{styling.cardContent}` ?

Reply:
i did it before we moved to module.css format.

Reply:
i changed it !